services/web/server/tests/unit/with_dbs/02/test_projects_handlers__open_close.py

```diff
@@ -494,6 +494,43 @@ async def test_open_project_with_small_amount_of_dynamic_services_starts_them_au
         mocked_director_v2_api["director_v2_api.run_dynamic_service"].reset_mock()
 
 
+@pytest.mark.parametrize(*standard_user_role())
+async def test_open_project_with_disable_service_auto_start_set_overrides_behavior(
+    client: TestClient,
+    logged_user: UserInfoDict,
+    user_project_with_num_dynamic_services: Callable[[int], Awaitable[ProjectDict]],
+    client_session_id_factory: Callable,
+    expected: ExpectedResponse,
+    mocked_director_v2_api: dict[str, mock.Mock],
+    mock_catalog_api: dict[str, mock.Mock],
+    max_amount_of_auto_started_dyn_services: int,
+    faker: Faker,
+):
+    assert client.app
+    num_of_dyn_services = max_amount_of_auto_started_dyn_services or faker.pyint(
```

Comment:
why not simply using the faker?

Reply:
because that max_amount_of_auto_started_dyn_services can be 0